{"status": "broken", "uuid": "b5aa8db5-7170-47cb-8ff6-460559da7504", "labels": [{"name": "suite", "value": "test_shopping_cart"}, {"name": "subSuite", "value": "TestShoppingCart"}, {"name": "parentSuite", "value": "automated-tests.page_object_pattern.tests"}, {"name": "host", "value": "azg-ThinkPad-T430"}, {"name": "thread", "value": "10153-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython2"}, {"name": "package", "value": "automated-tests.page_object_pattern.tests.test_shopping_cart"}], "stop": 1582403101445, "testCaseId": "2dc15cf59379efba804c006915f8e9c4", "historyId": "5f89292e02448e5b2843459a16f256a1", "start": 1582403101445, "fullName": "automated-tests.page_object_pattern.tests.test_shopping_cart.TestShoppingCart#test_proceed_to_checkout_logged_in", "statusDetails": {"message": "TypeError: super() takes at least 1 argument (0 given)", "trace": "request = <SubRequest 'setup' for <Function test_proceed_to_checkout_logged_in>>\n\n    @pytest.fixture()\n    def setup(request):\n>       driver = webdriver.Chrome(ChromeDriverManager().install())\n\nautomated-tests/conftest.py:11: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/usr/local/lib/python2.7/dist-packages/webdriver_manager/chrome.py:24: in install\n    driver_path = self.download_driver(self.driver)\n/usr/local/lib/python2.7/dist-packages/webdriver_manager/manager.py:34: in download_driver\n    cached_path = self.driver_cache.find_file_if_exists(driver.get_name(), driver.get_os_type(),\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <webdriver_manager.driver.ChromeDriver object at 0x7f72c03d2c90>\n\n    def get_os_type(self):\n>       if \"win\" in super().get_os_type():\nE       TypeError: super() takes at least 1 argument (0 given)\n\n/usr/local/lib/python2.7/dist-packages/webdriver_manager/driver.py:48: TypeError"}, "name": "test_proceed_to_checkout_logged_in"}